xbmc/cores/VideoPlayer/DVDDemuxers/DemuxMultiSource.h

```diff
@@ -45,22 +45,21 @@ class CDemuxMultiSource : public CDVDDemux
 public:
   CDemuxMultiSource();
   ~CDemuxMultiSource() override;
-  
+
+  bool Open(CDVDInputStream* pInput);
+
+  // implementation of CDVDDemux
   void Abort() override;
   void EnableStream(int64_t demuxerId, int id, bool enable) override;
   void Flush() override;
-  std::string GetFileName() override { return ""; };
   int GetNrOfStreams() const override;
-  CDemuxStream* GetStream(int iStreamId)const override { return nullptr; } ;
```

Comment:
Uh oh, this doesn't look good. In the superclass this is pure-virtual. I think I moved this to `protected` access but it got lost in the rebase. I'll try running a test build and see if I'm correct.